src/main/java/StringCalculator.java

```diff
@@ -0,0 +1,64 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class StringCalculator {
+
+    List<Integer> numbers = new ArrayList<>();
+    List<String> operators = new ArrayList<>();
+
+    // "1 + 2 * 3 / 1"
+    // 수식이 들어오면 계산하고 결과 반환
+    public int calculate(String expression) {
+        if (isNullOrEmpty(expression)) throw new IllegalArgumentException("입력 값이 null 혹은 빈 문자열입니다.");
+
+        int result = 0;
+
+        String[] strings = expression.split(" ");
+
+        for (int i = 0; i < strings.length; i++) {
+            if (i % 2 == 0) {
+                numbers.add(Integer.valueOf(strings[i]));
+            } else {
+                operators.add(strings[i]);
+            }
+        }
+
+        if (!isOperator(operators)) throw new IllegalArgumentException("잘못된 사칙연산 기호입니다");
```

Comment:
`if`문은 단일 행으로 구성되어있더라도 다음과 같이 쓰는 것을 권장함.
```suggestion
        if (!isOperator(operators)) {
          throw new IllegalArgumentException("잘못된 사칙연산 기호입니다");
        }
```